test/react-svg-path-precision.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+
+const {rules} = require("../lib/index.js");
+const RuleTester = require("eslint").RuleTester;
+
+const parserOptions = {
+    parser: "babel-eslint",
+};
+
+const ruleTester = new RuleTester(parserOptions);
+const rule = rules["react-svg-path-precision"];
+
+ruleTester.run("react-svg-path-precision", rule, {
+    valid: [
+        {
+            code: '<path d="M1.23,0.45L-0.98,42Z"/>',
+            options: [],
+        },
+        {
+            code: '<div><path d="M1.23,0.45Z"/><path d="M1.23,0.45Z"/></div>',
+            options: [],
+        },
+        {
+            code: '<path d="M1.234,0.456L-0.987,42Z"/>',
+            options: [{precision: 3}],
+        },
+    ],
+    invalid: [
+        {
+            code: '<path d="M1.234,0.456L-0.987,42Z"/>',
+            options: [],
+            errors: [
+                "This path contains numbers with too many decimal places.",
+            ],
+            output: "<path d=M1.23,0.45L-0.98,42Z/>",
```

Comment:
It feels like we should be rounding the numbers - for example `0.45` here should probably be `0.46` and `0.98` should be `0.99`.

Reply:
Maybe.  I opted to truncate to avoid having to do too much work.  I could make it configurable as to wether we truncate or round.

Reply:
I change to rounding and then, if it's too slow, re-evaluate truncation.